service/src/test/java/bio/terra/pipelines/testutils/MockMvcUtils.java

```diff
@@ -33,6 +33,8 @@ public static MockHttpServletRequestBuilder addJsonContentType(
   public static String convertToJsonString(Object obj) throws JsonProcessingException {
     ObjectMapper mapper = new ObjectMapper();
     mapper.configure(SerializationFeature.WRAP_ROOT_VALUE, false);
+    // needed because we currently allow amorphous objects currently as pipelineInputs
+    mapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
```

Comment:
i dont love that I did this but I think once we are validating inputs to maybe non amorphous classes we can get rid of it.  Open to any suggestions though

Reply:
hm. would it help to make it a key-value-pair structure, like a HashMap? (dictionary in Python)